File: ruby/lib/svix/api/operational_webhook.rb
# frozen_string_literal: true
# This file is @generated

require "net/http"

module Svix
  class OperationalWebhook
    attr_accessor :endpoint
    def initialize(client)
      @client = client
      @endpoint = OperationalWebhookEndpoint.new(client)
    end
  end
end

File: csharp/Svix/Api/OperationalWebhook.cs
// this file is @generated
#nullable enable
using Microsoft.Extensions.Logging;
using Svix.Models;

namespace Svix.Api
{
    public class OperationalWebhook(SvixClient client)
    {
        readonly SvixClient _client = client;

        public OperationalWebhookEndpoint Endpoint
        {
            get => new OperationalWebhookEndpoint(_client);
        }
    }
}
